lib/shoryuken/client.rb

```diff
@@ -9,15 +9,15 @@ def queues(name)
       end
 
       def sns
-        @sns ||= Aws::SNS::Client.new(aws_client_options(:sns_endpoint))
+        @sns ||= Aws::SNS::Client.new(aws_client_options(:sns))
       end
 
       def sns_arn
         @sns_arn ||= SnsArn
       end
 
       def sqs
-        @sqs ||= Aws::SQS::Client.new(aws_client_options(:sqs_endpoint))
+        @sqs ||= Aws::SQS::Client.new(aws_client_options(:sqs))
```

Comment:
@phstc I am passing no logfile to Shoryuken. `debug.log` and `info.log` are logs produced by Ruby on Rails. I assume Shoryuken is tied into their logging with this code, which I mention in our other conversion thread;

```
Shoryuken.configure_server do |config|
  Rails.logger = Shoryuken::Logging.logger
end
```